src/main/resources/messages.properties

```diff
@@ -1,21 +1,27 @@
-typeMismatch.courseConfigurationForm.totalClassMinutes = Must be a valid number.
-Min.courseConfigurationForm.totalClassMinutes = Must be greater than 0.
+typeMismatch.courseConfigurationForm.totalClassMinutes=Must be a valid number.
+Min.courseConfigurationForm.totalClassMinutes=Must be greater than 0.
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge keys "Min.courseConfigurationForm.defaultMinutesPerSession, Min.courseConfigurationForm.totalClassMinutes, Min.makeupForm.entries.minutesMadeUp, Min.rosterForm.sectionModels.attendances.minutesMissed" that have the same value "Must be greater than 0.". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=jproperties%3Aduplicated-values)